src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -57,6 +57,58 @@ public virtual FormulaValue GetResult(IRContext irContext)
             }
         }
 
+        private class VarianceAgg : IAggregator
+        {
+            protected int _count;
+            protected double _meanAcc;
+            protected double _m2Acc;
+
+            // Implementation of Welford's Algorithm:  https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance
+
+            public void Apply(FormulaValue value)
+            {
+                if (value is BlankValue)
+                {
+                    return;
+                }
+
+                var n1 = (NumberValue)value;
+
+                _count++;
+                var delta = n1.Value - _meanAcc;
+                _meanAcc += delta / _count;
+                var delta2 = n1.Value - _meanAcc;
+                _m2Acc += delta * delta2;
+            }
+
+            public virtual FormulaValue GetResult(IRContext irContext)
+            {
+                if (_count == 0)
+                {
+                    return CommonErrors.DivByZeroError(irContext);
+                }
+                else
+                {
+                    return new NumberValue(irContext, _m2Acc / _count);
```

Comment:
Looks good!
I think NaN and +/- Inf as the result of the division should also return an error.

Reply:
Got it. Added the necessary checks.

Reply:
We have a function for this, it's a pretty common case. Consider writing the return statement as 
`return FiniteChecker(irContext, 0, new NumberValue(irContext, _m2Acc/_count))`

Reply:
Modified the function to use FiniteChecker !